opendbc/car/toyota/carstate.py

```diff
@@ -218,6 +222,7 @@ def get_can_parsers(CP):
         ("GEAR_PACKET_HYBRID", 60),
         ("SECOC_SYNCHRONIZATION", 10),
         ("GAS_PEDAL", 42),
+        ("PCM_CRUISE_4", 1),
```

Comment:
Match above suggestion:
```suggestion
        ("PCM_CRUISE_4", 1),
```

```suggestion
```